SPECS/gawk/gawk.spec

```diff
@@ -40,10 +40,10 @@ find %{buildroot} -type f -name "*.la" -delete -print
 # Skip the timeout test, which is unreliable on our (vm) build machines
 sed -i 's/ timeout / /' test/Makefile
 sed -i 's/ pty1 / /' test/Makefile
+sed -i 's/ pma / /' test/Makefile
+sed -i 's/ pma-tests / /' test/Makefile
+sed -i 's/ check-recursive / /' test/Makefile
 
-# Generate locale for `en_US.iso88591` which is required for ptest
-# Ideally it should have been present. Investigate if its a `chroot` only issue
-%{_sbindir}/locale-gen.sh
 make %{?_smp_mflags} check
 
 %post   -p /sbin/ldconfig
```

Comment:
might be able to remove these calls to ldconfig, since they appear to cause the circular dependency, and probably are no longer needed per: https://fedoraproject.org/wiki/Changes/Removing_ldconfig_scriptlets